040-Data-operations/160-image-transformations.mdx

```diff
@@ -420,10 +425,12 @@ https://us-east-1.storage.xata.sh/transform/width=100,height=100,fit=contain/4u1
   url="https://us-east-1.storage.xata.sh/4u1fh2o6p10blbutjnphcste94"
   options={{ height: 100, width: 100, fit: 'contain' }}
 />
-
+@snide: I'm not sure about these double examples. Options I see: Kill the second example, or make it more obvious why there are two examples. Maybe by pulling out the "Note that the rendered example retains the original aspect-ratio, but doesn't grow beyond the original size" into markdown?
 <TabbedCode tabs={['TypeScript', 'Python', 'URL']}>
```

Comment:
@snide: I'm not sure about these double examples. Options I see: Kill the second example, or make it more obvious why there are two examples. Maybe by pulling out the "Note that the rendered example retains the original aspect-ratio, but doesn't grow beyond the original size" into markdown?

Reply:
I'm not worried about this. For people working with images, seeing the actual transformation is very important, even when it results in similar looking output.